plugins/scaffolder-actions/scaffolder-backend-module-http-request/src/actions/run/backstageRequest.test.ts

```diff
@@ -13,17 +13,35 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-import { createHttpBackstageAction } from './backstageRequest';
 import { Config, ConfigReader } from '@backstage/config';
 import os from 'os'; // eslint-disable-line
 import { getVoidLogger } from '@backstage/backend-common';
 import { PassThrough } from 'stream'; // eslint-disable-line
-import { http } from './helpers';
+import { createHttpBackstageAction } from './backstageRequest';
+import fetch from 'cross-fetch';
+
+jest.mock('cross-fetch');
```

Comment:
I started by trying to mock HttpClient but then gave up because jest was breaking my heart.